websites/P/Paramount+/presence.ts

```diff
@@ -9,11 +9,21 @@ const presence = new Presence({
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
-let title: string,
-	seasonEpi: string,
-	movTitle: string,
-	vidMdTl: Element,
-	oldUrl: string;
+let title: string, seasonEpisode: string, liveTitle: string;
+
+function pathIncludes(path: string, str: string) {
+	return path.includes(str);
+}
+
+const enum Logos {
+	Paramount = "https://i.imgur.com/VWrl85V.jpg",
+	CBS = "https://i.imgur.com/BfxpVej.png",
+	BET = "https://i.imgur.com/SBl99Dz.png",
+	ComedyCentral = "https://i.imgur.com/936hLEE.png",
+	MTV = "https://i.imgur.com/uCpqd3d.png",
+	Nickelodeon = "https://i.imgur.com/SwkpkrM.png",
+	Smithsonian = "https://i.imgur.com/1TfaVpT.png",
```

Comment:
All images to be used as `largeImageKey` or `smallImageKey` must be 512x512px exactly.